README.md

```diff
@@ -1,14 +1,196 @@
-Voice Quickstart for Android
-===
+# Twilio Voice Quickstart for Android
 
-This repository contains a simple Android application that exercises the incoming and outgoing call functionality of the [Twilio Programmable Voice SDK](https://www.twilio.com/docs/api/voice-sdk).
+Get started with Voice on Android:
 
-It is intended to be used in conjunction with the server-side component available at [https://github.com/twilio/voice-quickstart-server-python](https://github.com/twilio/voice-quickstart-server-python).
+- [Quickstart](#quickstart) - Run the quickstart app
+- [Reducing APK Size](#reducing-apk-size) - Use ABI splits to reduce APK size
+- [More Documentation](#more-documentation) - More documentation related to the Video Android SDK
+- [Issues & Support](#issues-and-support) - Filing issues and general support
 
-Please see the [Getting Started Guide](https://www.twilio.com/docs/api/voice-sdk/android/getting-started) for step-by-step instructions for installing and running the Quickstart app.
+## Quickstart
 
-For detailed documentation of the Twilio Programmable Voice Android SDK please see the [API Docs](https://media.twiliocdn.com/sdk/android/voice/latest/docs).
+To get started with the Quickstart application follow these steps. Steps 1-6 will allow you to make a call. The remaining steps 7-8 will enable push notifications using GCM.
 
-License
----
+
+1. [Open this project in Android Studio](#bullet1)
+2. [Create a Voice API key] (#bullet2)
+3. [Configure a server to generate an access token to use in the app](#bullet3)
+4. [Create a TwiML application](#bullet4)
+5. [Configure your application server](#bullet5)
+6. [Run the app](#bullet6)
+7. [Generate google-services.json from the Google Developer Console](#bullet7)
+8. [Add a Push Credential using your GCM `Server API Key`](#bullet8)
+9. [Receiving an Incoming Notification](#bullet9)
+
+
+### <a name="bullet1"></a>1. Open the project in Android Studio
+<img width="700px" src="images/quickstart/import_project.png"/>
+
+For now you will see an error in the console stating `Failed to retrieve GCM token`  and this is because you don’t have a valid `google-services.json` file yet to enable push notifications. We will obtain it in step 7.
+
+<img width="700px" src="images/quickstart/invalid_sender.png"/>
+
+### <a name="bullet2"></a>2. Create a Voice API key
+
+Go to the API Keys page and create a new API key
```

Comment:
Would it be a good idea to put the console link here? the link format may change though but would make it easier for the developers.